protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
+  optional string name = 1;
+  repeated common.MajorType major_type = 2;
```

Comment:
What does major type mean for a function? Is this the parameter type? The return type? Something else? Perhaps choose a clearer name or include a comment to explain the meaning.